lib/converter.js

```diff
@@ -1,13 +0,0 @@
-var xmlNodes = require('xml-nodes');
-var xmlObjects = require('xml-objects');
-var pumpify = require('pumpify');
```

Comment:
these packages are used elsewhere or can be removed from the package.json?

Reply:
I will check.